openslides/assignment/models.py

```diff
@@ -15,24 +15,36 @@
 from openslides.utils.exceptions import OpenSlidesError
 from openslides.utils.models import AbsoluteUrlMixin
 from openslides.utils.rest_api import RESTModelMixin
-from openslides.utils.utils import html_strong
 from openslides.users.models import User
 
 
-class AssignmentCandidate(RESTModelMixin, models.Model):
+class AssignmentRelatedUser(RESTModelMixin, models.Model):
     """
-    Many2Many table between an assignment and the candidates.
+    Many to Many table between an assignment and user.
     """
-    assignment = models.ForeignKey("Assignment")
-    person = models.ForeignKey(User, db_index=True)
-    elected = models.BooleanField(default=False)
-    blocked = models.BooleanField(default=False)
+    STATUS_CANDIDATE = 1
+    STATUS_ELECTED = 2
+    STATUS_BLOCKED = 3
+    STATUSES = (
+        (STATUS_CANDIDATE, ugettext_lazy('candidate')),
+        (STATUS_ELECTED, ugettext_lazy('elected')),
+        (STATUS_BLOCKED, ugettext_lazy('blocked')),
+    )
+
+    assignment = models.ForeignKey(
+        "Assignment",
+        db_index=True,
+        related_name='assignment_related_users')
+    user = models.ForeignKey(User, db_index=True)
+    status = models.IntegerField(
+        choices=STATUSES,
+        default=STATUS_CANDIDATE)
 
     class Meta:
-        unique_together = ("assignment", "person")
+        unique_together = ('assignment', 'user')
 
     def __str__(self):
-        return str(self.person)
+        return "%s <-> %s" % (self.assignment, self.user)
```

Comment:
Hm. I don't agree with this as a styling rule. There may be many reasons to do it this or that way (E. g. as a righty I like typing single quotes only with the right hand on standard German keyboard.) but I think most of the code written by @ostcar and me is with mainly single quotes everywhere so we can keep this way ...